spec/cli_spec.rb

```diff
@@ -0,0 +1,175 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Policial::CLI do
+  subject { described_class.new }
+
+  before(:each) { ENV['GITHUB_ACCESS_TOKEN'] = '1234' }
+  after(:each) { ENV['GITHUB_ACCESS_TOKEN'] = nil }
+
+  describe '#investigate' do
+    context 'when GITHUB_ACCESS_TOKEN environment variable is not set' do
+      it 'aborts with exit status code 1' do
+        ENV['GITHUB_ACCESS_TOKEN'] = nil
+
+        output = capture(:stderr) do
+          expect(-> { subject.investigate('volmer/policial', 1) })
+            .to exit_with_code(1)
+        end
+
+        expect(output).to eq(
+          "You must set the GITHUB_ACCESS_TOKEN environment variable.\n"
+        )
+      end
+    end
+
+    context 'when repo is not specified' do
```

Comment:
In RSpec I try to avoid `context` blocks that doesn't contain `before` or `let` blocks. Descriptions can be included in the right in the examples.